src/Core/Client.vala

```diff
@@ -144,8 +144,7 @@ public class AppCenterCore.Client : Object {
                 refresh_in_progress = true;
                 updating_cache = true;
                 try {
-                    success = yield PackageKitBackend.get_default ().refresh_cache (cancellable);
-                    yield FlatpakBackend.get_default ().refresh_cache (cancellable);
+                    success = yield BackendAggregator.get_default ().refresh_cache (cancellable);
```

Comment:
Was previously doing this two backends separately because the Ubuntu drivers one didn't really have a concept of `refresh_cache`, but it now has caching, so may as well call `refresh_cache` on all backends now.